api/filters/paging.go

```diff
@@ -52,22 +52,32 @@ func (*pagingFilter) Name() string {
 func (pf *pagingFilter) Run(req *web.Request, next web.Handler) (*web.Response, error) {
 	ctx := req.Context()
 	maxItems := req.URL.Query().Get("max_items")
-	limit, err := pf.validateMaxItemsQuery(maxItems)
+	limit, err := pf.parseMaxItemsQuery(maxItems)
```

Comment:
currently there is also an antlr the idea of which is to translate rest queries to criterions - are we going to try to include the paging query parsing there too ? If so we should probably consider this while implementing the paging (for example if we wanted to reuse the antlr stuff, we would probably need a query paramater `pagingQuery=limit=5 and token=abc` and introduce a PagingCriteria type. (there are other approaches as well..)

Reply:
We must clarify which is going first and which needs to be adopted because there will be conflicts for sure.